screeps-start.js

```diff
@@ -21,6 +21,16 @@ const loadPackage = (dir) =>
 const isDependency = (pkg, [name, version]) =>
   pkg.includes(name) || version.includes(pkg);
 
+const parseVersionSpec = (spec) => {
+  const atIdx = spec.lastIndexOf("@");
+  if (atIdx === -1) {
+    return [spec, "latest"];
+  }
```

Comment:
How does this handle scoped packages without a version? Say I had `@screeps/simplemod` without a version tag?
I think a naive solution would be to check if there is a `/` after the `@`?

Reply:
Right, I forgot about those. I guess a `atIdx !== 0` would work? I'm not exceptionally knowledgeable in npm's package name formats.

Or maybe write some sort of version parser and check the ending part for being one, but I really didn't feel like it was needed.